core/controllers/classifier.py

```diff
@@ -133,11 +135,11 @@ class NextJobHandler(base.BaseHandler):
     REQUIRE_PAYLOAD_CSRF_CHECK = False
 
     @acl_decorators.open_access
-    def post(self):
```

Comment:
This was meant to be `GET`. How did we miss this? The Oppia-ml Extension describes this as a GET method.

Reply:
Actually I think if it changes status of models in backend, it should be a POST...

Reply:
Reverted to POST. I'll change Oppia-ml Extension accordingly.